pkg/cmd/update-repositories/update_repositories.go

```diff
@@ -0,0 +1,69 @@
+package update_repositories
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/coreeng/corectl/pkg/git"
+	// "github.com/coreeng/corectl/pkg/logger"
+	// "go.uber.org/zap"
+)
+
+func UpdateRepositories(config *config.Config) error {
```

Comment:
New, this should be called by any command that needs the repos to be up to date.